### DISCUSSION TITLE
Help with virion detection

### DISCUSSION THREAD

**Discussion by jakecroft** [nextpyp/.github#107] [Q&A] [answered]
Hello, I have a dataset with some enveloped viruses. I tried running auto virion detection, but they are not perfectly spherical and have some heterogeneity in size so I had little success getting the auto virion detection to find all of them. Therefore, I manually picked the virion centers and ran virion detection with "manual" method specified, and now the list called "Auto Virions" seems to accurately have selected the virion centers when I look in the reconstruction tab. 

However, when I go to the segmentation tab it says "Showing X virions from manual_virions", and while it says it has the correct number of virions I see in the reconstruction tab, none of the images it shows seem to be over virions or correspond to the Auto virions points or the manually selected points.  I also can't change which list (manual or auto) it is showing. Since the virions have a range of sizes, I wanted to make sure they were segmented properly before running spike detection, but the segmentation tab has me concerned something is wrong.

**Comment by abartesaghi** [comment, marked as answer]
Are you using a `Binning factor for reconstruction` that is not the default value of 8? If so, this could be due to a bug in the code where we are not properly accounting for different levels of binning when picking virions manually. A temporary workaround would be to set the binning factor for reconstruction to 8 (your list of virions will still contain the correct coordinates, so you don't need to change that). We will fix this in the next release. Thanks for reporting.

Regarding the list names, the segmentation will only be done for the list of positions that were selected/calculated when running the pre-processing block. In your case, that would be the manual list. Also, the positions in your manual list will be copied to the Auto Virions list once the block is ran, so they will contain the same set of coordinates.

**Reply by jakecroft** [reply]
Ahh yep, thank you, that is what it was. I was using a higher binning factor.  Thanks for figuring it out!
